package.json

```
@@ -29,95 +29,104 @@
+    "redux-observable-es6-compat": "1.2.1",
+    "rxjs": "6.5.5",
+    "rxjs-compat": "6.5.5",
+    "semantic-release": "17.1.1",
+    "semantic-release-cli": "5.3.1"
   },
   "devDependencies": {
-    "@babel/core": "7.10.1",
+    "@babel/core": "7.10.3",
     "@babel/preset-react": "7.10.1",
-    "@commitlint/cli": "^8.3.5",
-    "@commitlint/config-conventional": "^8.3.4",
-    "@nrwl/cypress": "8.11.2",
-    "@nrwl/eslint-plugin-nx": "8.11.2",
-    "@nrwl/express": "8.11.2",
-    "@nrwl/jest": "8.11.2",
-    "@nrwl/node": "8.11.2",
-    "@nrwl/react": "^8.11.2",
-    "@nrwl/storybook": "^9.2.1",
-    "@nrwl/web": "8.11.2",
-    "@nrwl/workspace": "8.11.2",
-    "@storybook/addon-knobs": "5.3.9",
-    "@storybook/angular": "5.3.9",
-    "@storybook/react": "5.3.9",
-    "@testing-library/react": "9.4.0",
-    "@types/express": "4.17.2",
-    "@types/jest": "24.9.0",
-    "@types/lodash.camelcase": "^4.3.6",
-    "@types/node": "~13.1.8",
-    "@types/react": "16.9.19",
-    "@types/react-dom": "16.9.5",
-    "@typescript-eslint/eslint-plugin": "2.17.0",
-    "@typescript-eslint/parser": "2.17.0",
+    "@commitlint/cli": "9.0.1",
+    "@commitlint/config-conventional": "9.0.1",
+    "@nrwl/cypress": "9.5.0",
+    "@nrwl/eslint-plugin-nx": "9.5.0",
+    "@nrwl/express": "9.5.0",
+    "@nrwl/jest": "9.5.0",
+    "@nrwl/node": "9.5.0",
+    "@nrwl/react": "9.5.0",
+    "@nrwl/storybook": "9.5.0",
+    "@nrwl/web": "9.5.0",
+    "@nrwl/workspace": "9.5.0",
+    "@storybook/addon-knobs": "5.3.19",
+    "@storybook/angular": "5.3.19",
+    "@storybook/react": "5.3.19",
+    "@testing-library/react": "10.4.3",
+    "@types/express": "4.17.6",
+    "@types/jest": "26.0.3",
+    "@types/lodash.camelcase": "4.3.6",
+    "@types/node": "14.0.14",
+    "@types/react": "16.9.41",
+    "@types/react-dom": "16.9.8",
+    "@typescript-eslint/eslint-plugin": "3.5.0",
+    "@typescript-eslint/parser": "3.5.0",
     "babel-loader": "8.1.0",
     "dotenv": "8.2.0",
-    "eslint": "6.8.0",
-    "eslint-config-prettier": "6.9.0",
-    "eslint-plugin-import": "2.20.0",
-    "eslint-plugin-jsx-a11y": "6.2.3",
-    "eslint-plugin-react": "7.18.0",
-    "eslint-plugin-react-hooks": "2.3.0",
-    "jest": "25.1.0",
-    "prettier": "1.19.1",
-    "serverless-dotenv-plugin": "^2.3.2",
-    "serverless-finch": "^2.5.2",
-    "ts-jest": "25.0.0",
-    "ts-node": "~8.6.2",
-    "tslint": "~6.0.0",
-    "typescript": "~3.7.5",
-    "webpack-cli": "^3.3.10"
+    "eslint": "7.3.1",
+    "eslint-config-prettier": "6.11.0",
+    "eslint-plugin-import": "2.22.0",
+    "eslint-plugin-jsx-a11y": "6.3.1",
+    "eslint-plugin-react": "7.20.2",
+    "eslint-plugin-react-hooks": "4.0.4",
+    "jest": "26.1.0",
+    "prettier": "2.0.5",
+    "serverless-dotenv-plugin": "2.4.2",
+    "serverless-finch": "2.6.0",
+    "ts-jest": "26.1.1",
+    "ts-node": "8.10.2",
+    "tslint": "6.1.2",
+    "typescript": "3.9.5",
+    "webpack-cli": "3.3.12",
+    "eslint-plugin-cypress": "^2.11.1"
```

Can you install this again through npm so it locate it properly?
```suggestion
    "eslint-plugin-cypress": "2.11.1"
```

---

done 👍